api/v1alpha1/amaltheasession_children.go

```diff
@@ -201,6 +201,17 @@ func (cr *AmaltheaSession) sessionLocalhostURL() *url.URL {
 	return &output
 }
 
+// The localhost path prefix URL for the session. It excludes the auth proxy and the ingress and
+// the host is always 127.0.0.1.
+func (cr *AmaltheaSession) localhostPathPrefixURL() *url.URL {
+	if cr.Spec.Ingress.PathPrefix == "" {
+		return cr.sessionLocalhostURL()
+	}
```

Comment:
This branch sounds like it would make it very easy to not understand how to properly configure an Amalthea Session.

If the default URL for my session is `/lab`, then the ingress will fully restrict the user to `/lab` URLs, but actually, some other paths may be valid, e.g `/tree` and `/notebooks` are valid paths when jupyter server is running.

Reply:
Yeah I can make it `/`.